src/sub_platforms/sql_opt/videx/videx_service.py

```diff
@@ -30,6 +31,57 @@
 app = Flask(__name__)
 ENV_KEY_POST_VIDEX_META = 'POST_VIDEX_META'
 
+# Create API object
+api = Api(
+    app,
+    version='1.0',
+    title='VIDEX API',
+    description='This API doc integrates Swagger support into the VIDEX server using the Flask-RESTX, providing automated API description and interactive testing.',
+    doc='/docs',
+    ui=True,
+    validate=True
+)
+
+# Define a single namespace to maintain original path compatibility
+ns = api.namespace('', description='VIDEX API List')
+
+# Standard response model
+response_model = api.model('Response', {
+    'code': fields.Integer(required=True, description='Status Code'),
+    'message': fields.String(required=True, description='Response Message'),
+    'data': fields.Raw(description='Response Data')
+})
+
+task_meta_model = api.model('TaskMeta', {
+    'task_id': fields.String(required=True, description='Task ID'),
```

Comment:
@YoungHypo 

In `TaskMeta` and `AskVidex`, the field "task_id" can be missing or nullable (e.g., `'task_id': null`). Could you please fix them and then retry the TPCH-Tiny example in `README.md`? 

FYI, I run `videx_build_env.py` will raise following exception:

```python
create_videx_env_multi_db(videx_env, meta_dict=meta_request.meta_dict, )
response = post_add_videx_meta(meta_request, videx_server_ip_port=videx_server_ip_port, use_gzip=True)
assert response.status_code == 200, response.text

AssertionError: {"errors": {"task_id": "None is not of type 'string'"}, "message": "Input payload validation failed"}
```